examples/complete/main.tf

```diff
@@ -13,21 +13,19 @@ module "resource_group" {
 # Observability Instances (Sysdig + AT)
 ##############################################################################
 
-# Create Sysdig and Activity Tracker instance
+# Create Sysdig
 module "observability_instances" {
-  source                         = "git::https://github.com/terraform-ibm-modules/terraform-ibm-observability-instances?ref=v1.1.1"
-  region                         = var.region
-  resource_group_id              = module.resource_group.resource_group_id
-  activity_tracker_instance_name = "${var.prefix}-at"
-  sysdig_instance_name           = "${var.prefix}-sysdig"
-  activity_tracker_plan          = "7-day"
-  sysdig_plan                    = "graduated-tier"
-  enable_platform_logs           = false
-  enable_platform_metrics        = false
-  logdna_provision               = false
-  activity_tracker_tags          = var.resource_tags
-  logdna_tags                    = var.resource_tags
-  sysdig_tags                    = var.resource_tags
+  source                     = "git::https://github.com/terraform-ibm-modules/terraform-ibm-observability-instances?ref=v1.1.1"
+  region                     = var.region
+  resource_group_id          = module.resource_group.resource_group_id
+  sysdig_instance_name       = "${var.prefix}-sysdig"
+  sysdig_plan                = "graduated-tier"
+  enable_platform_logs       = false
+  enable_platform_metrics    = false
+  logdna_provision           = false
+  activity_tracker_provision = false
```

Comment:
the example still needs to support create a new activity tracker instance. So you need logic to mark this as true if no value is passed for `activity_tracker_crn`